app/models/beer.rb

```diff
@@ -24,4 +24,12 @@ def self.search_message
       "No Matching Beer"
     end
   end
+
+  def average_rating
+    if reviews.average(:rating) == nil
+      reviews.average(:rating)
```

Comment:
The problem was that if there were no reviews/ratings for a beer, then I would get an error, because there was no number to round. What I did works, but it is confusing, so I'm just having the 'if' return nil now.